app/models/concerns/callable.rb

```diff
@@ -0,0 +1,29 @@
+module Callable
+  extend ActiveSupport::Concern
```

Comment:
Yeah, that's right -- stuff pertaining to parsing calls or contacting patients goes here. I'll add a comment to reflect.